jsk_unitree_robot/cross/install.sh

```diff
@@ -54,7 +54,7 @@ function copy_data () {
     fi
 
     # Check if robot is reachable
-    reachability=$(ping -c4 ${hostname} 2>/dev/null | awk '/---/,0' | grep -Po '[0-9]{1,3}(?=% packet loss)')
+    reachability=$(ping -c4 ${hostname} 2>/dev/null | awk '/---/,0' | perl -nle'print $& while m{[0-9]{1,3}(?=% packet loss)}g')
```

Comment:
Is this for MacOS?

Reply:
If this change is for MacOS, I think it may be helpful to have a comment.

Reply:
Thanks for your comments.
It's for MacOS change.
I added a comment.
https://github.com/k-okada/jsk_robot/pull/66/commits/1c2cb4d1b02bf6d3aa873165d726aef66a23aace